src/pybind/mgr/dashboard/frontend/e2e/block/images.e2e-spec.ts

```diff
@@ -1,4 +1,5 @@
 import { Helper } from '../helper.po';
+import { PageHelper } from '../page-helper.po';
```

Comment:
Since `ImagePage` is extending `PageHelper` we can use `ImagePage` to call `PageHelper` methods.
Linter didn't complained about this and I'm guessing compilation will also not complain.

This also applies to all other files.

```suggestion
```

Reply:
Hum, no strong opinion, so I'm fine with it.